File: Android Curso Java/ConvertidorMonedas/app/src/main/java/com/example/convertidormonedas/MainActivity.java
package com.example.convertidormonedas;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    public void enviar(View view){
        float pesos;
        EditText editText = (EditText) findViewById(R.id.pesosEditText);
        TextView textView = (TextView) findViewById(R.id.dolaresTextView);

        pesos = (float) (Float.parseFloat(editText.getText().toString()) * 0.045);
        String myPesos = Float.toString(pesos);
        textView.setText(myPesos + " dolares");
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: Android Curso Java/ContadorKotlin/app/src/main/java/com/example/contadorkotlin/MainActivity.kt
package com.example.contadorkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var contador = 0

    fun incrementar(view: View){
        this.contador++
        textView.text = this.contador.toString()
    }

    fun reiniciar(view: View){
        this.contador = 0
        textView.text = this.contador.toString()
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var textView = findViewById<TextView>(R.id.textView)
    }
}